cms/djangoapps/export_course_metadata/tasks.py

```diff
@@ -6,14 +6,16 @@
 
 from celery import shared_task
 from django.core.files.base import ContentFile
+from edx_django_utils.monitoring import set_code_owner_attribute
 from opaque_keys.edx.keys import CourseKey
 from openedx.core.djangoapps.schedules.content_highlights import get_all_course_highlights
 
 from .storage import course_metadata_export_storage
 
 
-@shared_task(bind=True)
-def export_course_metadata_task(self, course_key_string):  # pylint: disable=unused-argument
+@shared_task
```

Comment:
I don't know enough about these task arguments. Can we keep this PR just to adding code ownership, and you could create another PR for any other changes you plan to make?

Reply:
Sure, I can add it to my list of errata.

For reference, here's the docs: https://docs.celeryq.dev/en/stable/userguide/tasks.html#bound-tasks